test/augmentation/test_augmentation.py

```diff
@@ -3521,6 +3522,34 @@ def test_module(self, device, dtype):
         pass
 
 
+class TestRandomClahe(BaseTester):
```

Comment:
maybe not needed to subclass `BaseTester` if you don't implement all the methods ? @johnnv1

Reply:
augmentations had a custom base test class which test a bunch of things -- @shijianjian can help here?